src/base58.cpp

```diff
@@ -84,44 +90,45 @@ static const int8_t mapBase58[256] = {
     return true;
 }
 
+auto ConvertToLongArray(const Span<const unsigned char>& input, size_t leadingZeros) -> std::vector<int64_t> {
```

Comment:
convert to base56 quickly before converting to base58 to minimize the inner quadratic loop